test/fixtures/groups.yml

```diff
@@ -0,0 +1,7 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  name: MyString
```

Comment:
@HacksForHugs  Try not to include auto-generated files. Remove if nothing useful is added